src/script/components/panel/enrichedFields.js

```diff
@@ -37,11 +37,16 @@ ko.components.register('enriched-fields', {
     this.richProfileRepository = resolve(graph.RichProfileRepository);
     this.fields = ko.observable([]);
 
-    this.richProfileRepository.getUserRichProfile(ko.unwrap(userId)).then(richProfile => {
-      if (richProfile.fields) {
-        this.fields(richProfile.fields);
-        onFieldsLoaded(this.fields());
-      }
-    });
+    this.richProfileRepository
+      .getUserRichProfile(ko.unwrap(userId))
+      .then(richProfile => {
+        if (richProfile.fields) {
+          this.fields(richProfile.fields);
+          onFieldsLoaded(this.fields());
+        }
+      })
+      .catch(() => {
+        this.fields([]);
```

Comment:
`this.fields` is already initialized with an empty array so is it necessary to set it again?

Reply:
If the error occurs in `onFieldsLoaded()` then it would be necessary I think.

Reply:
Yeah good point @ffflorian we don't want to change the fields back to the empty array if the parent function fails. 
I'll just put a `noop` function in the catch then.